src/main/java/org/folio/event/service/UserOutboxService.java

```diff
@@ -54,67 +54,76 @@ public UserOutboxService() {
    * @return future with integer how many records have been processed
    */
   public Future<Integer> processOutboxEventLogs(Vertx vertx, Map<String, String> okapiHeaders) {
+    logger.debug("processOutboxEventLogs:: Trying to process outbox event logs");
     String tenantId = TenantTool.tenantId(okapiHeaders);
     PostgresClient pgClient = PostgresClient.getInstance(vertx, tenantId);
     return pgClient.withTrans(conn -> lockRepository.selectWithLocking(conn, OUTBOX_LOCK_NAME, tenantId)
       .compose(retrievedCount -> outboxRepository.fetchEventLogs(conn, tenantId))
       .compose(logs -> {
         if (CollectionUtils.isEmpty(logs)) {
+          logger.warn("processOutboxEventLogs:: OutBoxEventLogs is empty");
           return Future.succeededFuture(0);
         }
 
-        logger.info("Fetched {} event logs from outbox table, going to send them to kafka", logs.size());
+        logger.info("processOutboxEventLogs:: Fetched {} event logs from outbox table, going to send them to kafka", logs.size());
         List<Future<Boolean>> futures = getKafkaFutures(logs, okapiHeaders);
         return GenericCompositeFuture.join(futures)
           .map(logs.stream().map(OutboxEventLog::getEventId).toList())
           .compose(eventIds -> {
             if (CollectionUtils.isNotEmpty(eventIds)) {
               return outboxRepository.deleteBatch(conn, eventIds, tenantId)
-                .onSuccess(rowsCount -> logger.info("{} logs have been deleted from outbox table", rowsCount))
+                .onSuccess(rowsCount -> logger.info("processOutboxEventLogs:: {} logs have been deleted from outbox table", rowsCount))
                 .onFailure(ex -> logger.error("Logs deletion failed", ex));
             }
+            logger.info("processOutboxEventLogs:: eventIds is empty");
             return Future.succeededFuture(0);
           });
       })
     );
   }
 
   public Future<Boolean> saveUserOutboxLogForCreateUser(Conn conn, User user, UserEvent.Action action, Map<String, String> okapiHeaders) {
-      return userTenantService.isConsortiaTenant(conn, okapiHeaders)
+    logger.debug("saveUserOutboxLogForCreateUser:: Trying to save UserOutBoxLog for create user action");
+    return userTenantService.isConsortiaTenant(conn, okapiHeaders)
         .compose(isConsortiaTenant -> {
           if (isConsortiaTenant && isStaffOrSystemUser(user)) {
             return saveUserOutboxLog(conn, user, action, okapiHeaders);
           }
-          return Future.succeededFuture();
+          logger.info("saveUserOutboxLogForCreateUser:: OutBoxLog was NOT saved because user is NOT belong to consortia tenant");          return Future.succeededFuture();
```

Comment:
return Future.succeededFuture(); should be on next line

Reply:
Thanks, fixed